.github/workflows/automate.yml

```diff
@@ -0,0 +1,16 @@
+name: Merge
+on:
+  [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: docker run -v $(pwd):/work --workdir /work centos:centos8 /bin/bash -c 'yum install -y rpm-build $(cat automation/build-artifacts.packages) && ./automation/build-artifacts-manual.sh'
+      - name: Upload artifacts
```

Comment:
let's not depend on the existing scripts, we are going to drop them once we stop using stdci. let's add the build code here.